src/engine/TransitivePath.cpp

```diff
@@ -187,7 +191,15 @@ size_t TransitivePath::getSizeEstimate() {
 }
 
 // _____________________________________________________________________________
-size_t TransitivePath::getCostEstimate() { return getSizeEstimate(); }
+size_t TransitivePath::getCostEstimate() {
+  auto costEstimate = getSizeEstimate();
```

Comment:
// We assume that the cost of computing the transitive path is proportional to the result size.